bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/handler/VolvoOnCallBridgeHandler.java

```diff
@@ -62,7 +63,8 @@ public void initialize() {
         ApiBridgeConfiguration configuration = getConfigAs(ApiBridgeConfiguration.class);
 
         try {
-            VocHttpApi vocApi = new VocHttpApi(configuration, gson, httpClient);
+            String clientName = ThingWebClientUtil.buildWebClientConsumerName(thing.getUID(), null);
+            VocHttpApi vocApi = new VocHttpApi(clientName, configuration, gson, httpClientFactory);
```

Comment:
Could be called like this is not changing anything in `VocHttpApi`.
```suggestion
            VocHttpApi vocApi = new VocHttpApi(configuration, gson, httpClientFactory.createHttpClient(clientName););
```